### ISSUE TITLE
Accordion: Add 'tab-opened' and 'tab-closed' event

### ISSUE THREAD

**Issue by jayzzer** [primefaces/primevue#3074]
### Describe the feature you would like to see added

It would be a pleasure to have some event that will emit after open/close animation of `AccordionTab`, maybe on `Transition` component `after-leave` and `after-enter` hooks.

**Comment by tugcekucukoglu** [comment]
`tab-open` and `tab-open` events are already exists. Could you check the documentation please?

**Comment by jayzzer** [comment]
> `tab-open` and `tab-open` events are already exists. Could you check the documentation please?

Right, I know about this events, but `tab-open` and `tab-close` events are called immediately before the animation of opening of the tabs, but I want events that are called after the full disclosure of the tabs (after animation of `opening`/`closing` ends)
